tools/frb_internal/lib/src/makefile_dart/release.dart

```diff
@@ -103,6 +103,11 @@ void _updateVersionInText() {
           '$x/rust/src/**',
           '$x/../src/**',
         ]),
+    ...kDartExampleTestFeaturePackages.expand((x) => [
```

Comment:
(same, maybe we can remove kDartExampleTestFeaturePackages variable and its special treatment)